series/templates/series/show_details.html

```diff
@@ -1,66 +1,71 @@
+{% extends 'series/base.html' %}
 {% load static %}
 
-<link rel="stylesheet" type="text/css" href="{% static 'series/show_details.css' %}">
-<link rel="stylesheet" type="text/css" href="{% static 'series/style.css' %}">
-<script type="text/javascript" src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
-<script type="text/javascript" src="{% static 'series/subscription.js' %}"></script>
-<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css"
-      integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
+{% block stylesheets %}
+    <link rel="stylesheet" type="text/css"
+          href="{% static 'series/show_details.css' %}">
+    <link rel="stylesheet"
+          href="https://use.fontawesome.com/releases/v5.4.1/css/all.css"
+          integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz"
+          crossorigin="anonymous">
+{% endblock %}
 
-<!--Navigation bar-->
-<ul class="navbar">
-    <li class="tab"><a href="{% url 'search_series' %}" id="active">Search</a></li>
-    <li class="tab"><a href="{% url 'followed_series' %}">Favorite shows</a></li>
-    <li class="tab"><a href="{% url 'profile' %}">Profile</a></li>
-    <li class="tab"><a href="{% url 'search_series' %}">About</a></li> <!--TODO link to about page-->
-</ul>
+{% block content %}
+    <h1>{{ show.title }}</h1>
```

Comment:
For some reason, it is not centered anymore, even though it is h1. do you override the style somewhere?

Reply:
Fixed!